http.proxy/zilla.json

```diff
@@ -9,7 +9,7 @@
             {
                 "trust":
                 {
-                    "store": "truststore.p12",
+                    "store": "/etc/zilla/ssl/truststore.p12",
```

Comment:
Paths are resolved relative to config location, so this would be better changed to a relative path `ssl/truststore.p12`. The details of inside the image should not leak out into the config file mounted from the example, so relative and relocatable is better than absolute paths.

Suggest creating a local directory in the example called `conf` containing `zilla.json` and subdirectory `tls` (not `ssl` as we use `tls` in the config), then a single volume mount for `./conf` locally to `/etc/zilla` in the image.